### PR TITLE
Adding sitemap index to the root file

### PR DESCRIPTION
**Pull request by wender**
#### What problem is this solving?
Make a list of stores available to the Search Engines

#### How to test it?

[Workspace](https://sitemap--sandboxusdev.myvtex.com/sitemap.xml)

#### Screenshots or example usage:
image

### CODE REVIEW

**Review by SDBowen — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -33,20 +33,24 @@ Once installed, the app will generate a new route called `/stores` for your stor
...
+| :------------------: | :------------------------------------------------------------------------------------------------: |
+|     `store-list`     |               Renders a list of retail stores and a map with all their localization.               |
+|    `store-group`     | Provides the pickup point data to other blocks exported by the app, such as the ones listed below. |
+|     `store-name`     |                                      Renders the store name.                                       |
+|  `store-back-link`   |                           Renders a link to return to the previous page.                           |
+|     `store-map`      |                         Renders a map with the retail store localization.                          |
+|   `store-address`    |                                    Renders the store's address.                                    |
+|    `store-hours`     |                                 Renders the store's opening hours.                                 |
+| `store-instructions` |                    Renders the desired instructions to access the retail store.                    |
```
Comment (guerreirobeatriz) [inline docs/README.md:53]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 2]
```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
...
-| `icon` | `string` | Icon URL (`svg` or `png`) to be displayed alongside with the `store-list` blocks. | Google's default. |
-| `iconWidth` | `number` | Icon width in pixels (`px`). | Image default width. |
-| `iconHeight` | `number` | Icon height in pixels (`px`). | Image default height. |
+|   Prop name   |   Type   |                                    Description                                    |     Default value     |
+| :-----------: | :------: | :-------------------------------------------------------------------------------: | :-------------------: |
+| `filterByTag` | `string` |      Desired tags to filter the Pickup Points fetched and displayed. You can      |      `undefined`      |
+|    `icon`     | `string` | Icon URL (`svg` or `png`) to be displayed alongside with the `store-list` blocks. |   Google's default.   |
+|  `iconWidth`  | `number` |                           Icon width in pixels (`px`).                            | Image default width.  |
+| `iconHeight`  | `number` |                           Icon height in pixels (`px`).                           | Image default height. |
```
Comment (guerreirobeatriz) [inline docs/README.md:156]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 3]
```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
...
-| `imageSelector` | `string` | CSS Selector that will wrap all the images displayed on the UI. | `undefined` |
-| `phoneSelector` | `string` | CSS Selector that will wrap all the phone numbers displayed on the UI. | `undefined` |
+|    Prop name    |   Type   |                                                          Description                                                           | Default value |
+| :-------------: | :------: | :----------------------------------------------------------------------------------------------------------------------------: | :-----------: |
+|     `title`     | `string` | Text to be displayed alongside with the store name. Use `{storeName}` as value to display the store's accurate name on the UI. | `{storeName}` |
+| `imageSelector` | `string` |                                CSS Selector that will wrap all the images displayed on the UI.                                 |  `undefined`  |
+| `phoneSelector` | `string` |                             CSS Selector that will wrap all the phone numbers displayed on the UI.                             |  `undefined`  |
```
Comment (guerreirobeatriz) [inline docs/README.md:164]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 4]
```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
...
 #### `store-name` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `text` | `string` | Text to be displayed alongside with the store name. Use `{storeName}` as value to display the store's accurate name on the UI. | `{storeName}` |
-| `tag` | `string` | HTML element to wrap the `store-name` block when rendered on the UI. | `div` |
+| Prop name |   Type   |                                                          Description                                                           | Default value |
+| :-------: | :------: | :----------------------------------------------------------------------------------------------------------------------------: | :-----------: |
+|  `text`   | `string` | Text to be displayed alongside with the store name. Use `{storeName}` as value to display the store's accurate name on the UI. | `{storeName}` |
+|   `tag`   | `string` |                              HTML element to wrap the `store-name` block when rendered on the UI.                              |     `div`     |
```
Comment (guerreirobeatriz) [inline docs/README.md:173]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 5]
```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
...
 
 #### `store-back-link` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `label` | `string` | Entitles the `store-back-link` block when rendered on the UI. | `Back to all stores` |
+| Prop name |   Type   |                          Description                          |    Default value     |
+| :-------: | :------: | :-----------------------------------------------------------: | :------------------: |
+|  `label`  | `string` | Entitles the `store-back-link` block when rendered on the UI. | `Back to all stores` |
```
Comment (guerreirobeatriz) [inline docs/README.md:179]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 6]
```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
...
 #### `store-map` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `width` | `string` | Map width in pixels (`px`). | `100%` |
-| `height` | `string` | Map height in pixels (`px`). | `200px` |
+| Prop name |   Type   |         Description          | Default value |
+| :-------: | :------: | :--------------------------: | :-----------: |
+|  `width`  | `string` | Map width in pixels (`px`).  |    `100%`     |
+| `height`  | `string` | Map height in pixels (`px`). |    `200px`    |
```
Comment (guerreirobeatriz) [inline docs/README.md:186]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 7]
```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
...
 
 #### `store-address` props
 
-| Prop name | Type | Description | Default value     |
-| :-------: | :--: | :---------: | :---------------: |
-| `label` | `string` | Entitles the `store-address` block when rendered on the UI. | `Store address` |
+| Prop name |   Type   |                         Description                         |  Default value  |
+| :-------: | :------: | :---------------------------------------------------------: | :-------------: |
+|  `label`  | `string` | Entitles the `store-address` block when rendered on the UI. | `Store address` |
```
Comment (guerreirobeatriz) [inline docs/README.md:192]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 8]
```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
...
 #### `store-hours` props
 
-| Prop name | Type | Description | Default value. |
-| :--------:| :--: | :---------: | :------------: |
-| `label` | `string` | Entitles the `store-hours` block when rendered on the UI. | `Store hours` |
-| `format` | `enum` | Hour format. Possible values are : `12h` and `24h`. | `24h` |
+| Prop name |   Type   |                        Description                        | Default value. |
+| :-------: | :------: | :-------------------------------------------------------: | :------------: |
+|  `label`  | `string` | Entitles the `store-hours` block when rendered on the UI. | `Store hours`  |
+| `format`  |  `enum`  |    Hour format. Possible values are : `12h` and `24h`.    |     `24h`      |
```
Comment (guerreirobeatriz) [inline docs/README.md:199]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 9]
```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
...
 
 #### `store-instructions` props
 
-| Prop name | Type | Description | Default value |
-| :------ : | :--: | :---------: | :-----------: |
-| `label` | `string` | Entitles the `store-instructions` block when rendered on the UI. | `Information` |
+| Prop name |   Type   |                           Description                            | Default value |
+| :-------: | :------: | :--------------------------------------------------------------: | :-----------: |
+|  `label`  | `string` | Entitles the `store-instructions` block when rendered on the UI. | `Information` |
```
Comment (guerreirobeatriz) [inline docs/README.md:205]: No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!

File: docs/README.md [hunk 10]
```diff
@@ -33,20 +33,24 @@ Once installed, the app will generate a new route called `/stores` for your stor
 
 The new page already contains a default template with all blocks exported by the Store Locator app, meaning that the `/stores` page is ready to be rendered and no further actions are required. However, you can **customize the new page overwriting the template by creating a brand new one as you wish**. To do so, check the **Advanced configurations** section below.
 
+This app will also add a new entry to the `/sitemap.xml` file to make all your Stores accessible to the Search Engines.
+
+Make sure you have the `vtex.store-sitemap@2.x` App installed on your account to make use of the sitemap feature.
```
Comment (guerreirobeatriz) [inline docs/README.md:38]:
```suggestion
> ℹ️ *This app will also **add a new entry to your store's `/sitemap.xml` file in order to have all the pickup points available to the search engines** - make sure you already have the `vtex.store-sitemap@2.x` app installed in your VTEX account!*
```

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

**Pull request merged**